docs/reference/api/storage/storage.md

```diff
@@ -63,6 +65,18 @@ We optimized this file system to work with a limited amount of RAM and ROM. It a
 
 The "little" in the little file system comes from the focus on both keeping resource usage low and keeping the scope self-contained. Aside from the three targeted issues above, there is a heavy restriction against bloat in this software module. Instead, we push additional features to separate layers in the BlockDevice API that drives the Mbed OS storage stack. This gives Mbed OS a tool for remaining flexible as technology used by IoT devices develops.
 
+#### The FATFileSystem
+
+The FAT file system is an established disk-oriented file system that you can find on Mbed OS, Windows, Linux and Mac OS X. Due to its age and popularity, the FAT file system has become the standard for portable storage, such as flash drives and SD cards.
+
+##### Portable
+
+The primary feature of the FAT file system is its portability. With support across PC operating systems, the FAT file system lets you access storage from both the embedded system your PC. This gives users a way to get information onto and off of the device.
```

Comment:
I've read some of this content three times now. Instead of duplicating it, let's keep API-specific content on the API reference pages and general concepts on the overview pages.

Reply:
I still think this causes problems for users, since the general concepts are important to understand the API and the users I've seen don't even see the overview page.

But feel free to update it so it's consistent.